tracer/src/Datadog.Trace/ContinuousProfiler/ContextTracker.cs

```diff
@@ -57,6 +59,14 @@ public void Set(ulong localRootSpanId, ulong spanId)
             WriteToNative(new SpanContext(localRootSpanId, spanId));
         }
 
+        public void SetEndpoint(ulong localRootSpanId, string endpoint)
+        {
+            if (IsEnabled && _isEndpointProfilingEnabled && !string.IsNullOrEmpty(endpoint))
```

Comment:
Is is possible that localRootSpanId is 0 but still valid?

Reply:
No, 0 is an invalid value for the span id